cypress/e2e/ScraperLiteEdition/Pricing.cy.js

```diff
@@ -0,0 +1,86 @@
+import { SCRAPER_BASE_URL } from '../../resources/data';
+
+
+describe('Scrapper(Lite Edition) Pricing page', () => {
+    before(() => {
+      cy.visit(SCRAPER_BASE_URL);
+    });
+  
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+  
+    it('2. Test to check "Scraper API (Lite Edition)" title is present', () => {
+        cy.componentVisiblityCheck('.h2', 'Scraper API (Lite Edition)');
+    });
+
+    it('3. Test to check "Free plan" is present', () => {
+        cy.componentVisiblityCheck(':nth-child(1) > .card > .card-header > .h3', 'Free Plan');
```

Comment:
Remove nth child all over the page in pricing page

Reply:
I have removed the nth child for some of the it blocks, and i kept nth child for the subscribe buttons as i'm checking the subscribe button visibility.